### PR TITLE
fix input radio id

### PR DESCRIPTION
**Pull request by ximex**

### CODE REVIEW

**Review by weeman1337 — CHANGES_REQUESTED** [review]

File: includes/sys_form.php [hunk 1]
```diff
@@ -148,10 +148,13 @@ function form_checkbox($name, $label, $selected, $value = 'checked', $html_id =
  */
 function form_radio($name, $label, $selected, $value)
 {
+    $value = htmlspecialchars((string)$value);
+    $id = $name . '_' . $value;
```
[the remark this answers is not in the document]
Comment (ximex) [reply includes/sys_form.php:152]: should i do a `.replace(' ', '-')`?

[the remark this answers is not in the document]
Comment (weeman1337) [reply includes/sys_form.php:152]: Doesn't catch all whitespaces. I would recommend something like this:

```
preg_replace('/\s/', '-', $id)
```

[the remark this answers is not in the document]
Comment (ximex) [reply includes/sys_form.php]: fixed

File: includes/pages/admin_shifts.php [hunk 2]
```diff
@@ -29,7 +29,7 @@ function admin_shifts()
     $session = session();
     $start = Carbon::createTimestampFromDatetime(date('Y-m-d') . 'T00:00');
     $end = $start;
-    $mode = '';
+    $mode = 'single';
```
Comment (MyIgel) [inline includes/pages/admin_shifts.php:32]: I would prefer `multi` here as normally before and on an event multiple shifts get created.

**Review by MyIgel — CHANGES_REQUESTED** [review]

File: includes/pages/admin_shifts.php (hunk 2 above)
Comment (ximex) [reply includes/pages/admin_shifts.php]: done

**Pull request merged**